modules/configs/files/install-ptfe.sh

```diff
@@ -9,6 +9,11 @@ if [ -s /etc/ptfe/proxy-url ]; then
   export http_proxy
   export https_proxy
   export no_proxy=10.0.0.0/8,127.0.0.1,169.254.169.254
+  if [[ $(< /etc/ptfe/repl-cidr) != "" ]]; then
+      repl_cidr=$(cat /etc/ptfe/repl-cidr)
+      export repl_cidr
+      export no_proxy=0.0.0.0/8,127.0.0.1,169.254.169.254,$repl_cidr
```

Comment:
since we already export no_proxy above does it make more sense to build this off the existing one?

also looks like there's a typo here on the 0.0.0.0/8 instead of the 10.0.0.0/8
`export no_proxy=10.0.0.0/8,127.0.0.1,169.254.169.254,$repl_cidr`
```suggestion
      export no_proxy=$no_proxy,$repl_cidr
```